app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person_data in people:
+        person_list.append(Person(person_data["name"], person_data["age"]))
+
+    for i in people:
+        person = Person.people[i["name"]]
+        if "wife" in i and i["wife"]:
```

Comment:
Use just dict.get() method in the condition, instead of this long one

Reply:
I see